src/storage.cc

```diff
@@ -318,7 +318,6 @@ void Storage::ProcessMat_(double cap) {
   using cyclus::Material;
   using cyclus::ResCast;
   using cyclus::toolkit::ResBuf;
-  using cyclus::toolkit::Manifest;
```

Comment:
While we're here... I don't see any reason for any of these `using` statements to be here...?

Reply:
Great point.  I went through `storage.cc` and cleaned up some other `using` statements.  A quick search of the cycamore archetypes shows there are a lot of `using` statements throughout the repo.  I will make an issue to go through and clean up where it is necessary